Bukkit/src/main/java/com/plotsquared/bukkit/generator/BukkitPlotGenerator.java

```diff
@@ -436,6 +436,9 @@ private final class BukkitPlotBiomeProvider extends BiomeProvider {
         static {
             ArrayList<Biome> biomes = new ArrayList<>(List.of(Biome.values()));
             biomes.remove(Biome.CUSTOM);
+            if (PlotSquared.platform().serverVersion()[1] <= 19) {
+                biomes.remove(Biome.CHERRY_GROVE);
+            }
```

Comment:
What is that supposed to do? Does folia not support cherry groves?

Reply:
Some 1.19.4 servers tries to generate plots without 1.20 datapacks.

Reply:
I don't understand the issue. CHERRY_GROVE exist in 1.19.4, what does Folia or 1.20 have to do with it?

Reply:
```
[13:21:06] [pool-13-thread-1/WARN]: [PlotSquared/BukkitPlatform] This message may also be a false positive and could be ignored.
[13:21:06] [Server thread/INFO]: [PlotSquared/BukkitPlatform] (UUID) 0 UUIDs will be cached
[13:21:06] [Server thread/INFO]: Preparing level "world"
[13:21:06] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.IllegalStateException: Missing key in ResourceKey[minecraft:root / minecraft:worldgen/biome]: ResourceKey[minecraft:worldgen/biome / minecraft:cherry_grove]
	at net.minecraft.core.Registry.getHolderOrThrow(Registry.java:162) ~[folia-1.19.4.jar:git-Folia-22]
	at java.util.Optional.orElseThrow(Optional.java:403) ~[?:?]
	at net.minecraft.core.Registry.getHolderOrThrow(Registry.java:162) ~[folia-1.19.4.jar:git-Folia-22]
	at org.bukkit.craftbukkit.v1_19_R3.block.CraftBlock.biomeToBiomeBase(CraftBlock.java:407) ~[folia-1.19.4.jar:git-Folia-22]
	at org.bukkit.craftbukkit.v1_19_R3.generator.CustomWorldChunkManager.biomeListToBiomeBaseList(CustomWorldChunkManager.java:28) ~[folia-1.19.4.jar:git-Folia-22]
	at org.bukkit.craftbukkit.v1_19_R3.generator.CustomWorldChunkManager.b(CustomWorldChunkManager.java:55) ~[folia-1.19.4.jar:git-Folia-22]
	at net.minecraft.world.level.biome.BiomeSource.d(BiomeSource.java:29) ~[?:?]
	at com.google.common.base.Suppliers$NonSerializableMemoizingSupplier.get(Suppliers.java:183) ~[guava-31.1-jre.jar:?]
	at net.minecraft.world.level.biome.BiomeSource.possibleBiomes(BiomeSource.java:40) ~[?:?]
	at net.minecraft.world.level.chunk.ChunkGeneratorStructureState.hasBiomesForStructureSet(ChunkGeneratorStructureState.java:157) ~[folia-1.19.4.jar:git-Folia-22]
	at net.minecraft.world.level.chunk.ChunkGeneratorStructureState.lambda$createForNormal$1(ChunkGeneratorStructureState.java:65) ~[folia-1.19.4.jar:git-Folia-22]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178) ~[?:?]
	at java.util.AbstractList$RandomAccessSpliterator.forEachRemaining(AbstractList.java:720) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682) ~[?:?]
	at net.minecraft.world.level.chunk.ChunkGeneratorStructureState.createForNormal(ChunkGeneratorStructureState.java:66) ~[folia-1.19.4.jar:git-Folia-22]
	at net.minecraft.world.level.chunk.ChunkGenerator.createState(ChunkGenerator.java:106) ~[folia-1.19.4.jar:git-Folia-22]
	at net.minecraft.server.level.ChunkMap.<init>(ChunkMap.java:305) ~[folia-1.19.4.jar:git-Folia-22]
	at net.minecraft.server.level.ServerChunkCache.<init>(ServerChunkCache.java:276) ~[?:?]
	at net.minecraft.server.level.ServerLevel.<init>(ServerLevel.java:645) ~[?:?]
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:616) ~[folia-1.19.4.jar:git-Folia-22]
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:465) ~[folia-1.19.4.jar:git-Folia-22]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:308) ~[folia-1.19.4.jar:git-Folia-22]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1191) ~[folia-1.19.4.jar:git-Folia-22]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:348) ~[folia-1.19.4.jar:git-Folia-22]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
[13:21:07] [Server thread/ERROR]: This crash report has been saved to: /Users/pglanz/development/workspace/gitlab_themeinerlp_dev/onelitefeather/PlotSquared/run-folia/./crash-reports/crash-2023-04-05_13.21.07-server.txt
[13:21:07] [Region shutdown thread/INFO]: [RegionShutdownThread] Awaiting scheduler termination for 60s
[13:21:07] [Region shutdown thread/INFO]: [RegionShutdownThread] Scheduler halted
[13:21:07] [Region shutdown thread/INFO]: Stopping server
```
Thats the generic issue on the main branch of PS. 
I prepared also already a non folia fix of it.